apps/guide/src/content/02-getting-started/03-adding-your-bot-to-a-server.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Adding your bot to a server
+category: Getting started
+---
+
+# Adding your bot to a server
+
+After you [set up an application](./setting-up-an-application), you'll notice it's not in any servers yet. So, how does that work?
+
+Before you're able to see your bot in a server, you will need to add it by using a unique invite link.
+
+## Bot invite links
+
+The basic version of one such link looks like this:
+
+<CH.Code lineNumbers={false}>
+
+```
+https://discord.com/api/oauth2/authorize?client_id=123456789012345678&permissions=0&scope=bot%20applications.commands
+```
+
+</CH.Code>
+
+The structure of the URL is quite simple:
+
+- _`https://discord.com/api/oauth2/authorize`_ is Discord's standard structure for authorizing an OAuth2 application (such as your bot application) for entry to a Discord server.
+- _`client_id=...`_ is to specify _which_ application you want to authorize. You'll need to replace this part with your client's id to create a valid invite link.
+- _`permissions=...`_ describes the permissions that your bot will request to be granted by default upon joining the server you are adding it to.
+- _`scope=bot%20applications.commands`_ specifies that you want to add this application as a Discord bot with the ability to create slash commands.
```

Comment:
```suggestion
- _`scope=bot`_ specifies that you want to add this application as a Discord bot with a presence in servers and the ability to receive gateway events.
```
those are basically mutually exclusive now, as the `bot` scope implies `applications.commands`. The focus of the `bot` scope is the presence in the server, enabling WS connections

Reply:
Should we strip `%20applications.commands` from the above URL or explain the inference?

Reply:
oh, i missed that, yes, can be removed.
if you want to cover the `application.commands` scope, I think an info box could work there
I do not remember if we have folding boxes that can be closed by default, or if I'm spending too much time in obsidian recently. that would be a nice use case, if we have that

Reply:
Went with removing it for now.